config.json

```diff
@@ -1128,12 +1128,12 @@
     },
     {
       "title": "Fast and lightweight",
-      "content": "Lua is known to be one of the fastest amongst interpreted dynamic languages and its interpreter, built with all standard libraries, takes under 300K.",
+      "content": "Lua is one of the fastest interpreted dynamic languages. Its compiler and interpreter fit in 300K.",
       "icon": ""
     },
     {
       "title": "Simple",
-      "content": "Lua has a simple, yet powerful syntax which completly fits into the head, which makes it easy and fast to learn.",
+      "content": "Lua has a simple, yet powerful syntax that is it easy and learn and master.",
```

Comment:
Little typo: `... that is it easy ...` to `... that is easy ...`

Reply:
Whoops, thanks for the catch!

Reply:
```suggestion
      "content": "Lua has a simple, yet powerful syntax that is it easy to learn and master.",
```